lms/djangoapps/certificates/tests/test_cert_management.py

```diff
@@ -11,7 +12,7 @@
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory, check_mongo_calls
 from student.tests.factories import UserFactory, CourseEnrollmentFactory
-from certificates.management.commands import resubmit_error_certificates, regenerate_user, ungenerated_certs
+from certificates.management.commands import resubmit_error_certificates, regenerate_user, ungenerated_certs, rescind_cert  # pylint: disable=line-too-long
```

Comment:
> there's got to be a way to import multiple functions from a file without having to disable a pylint thing...

The irony, of course, is that you literally ignore the long line by... making it even longer :P

This is one of the reasons I prefer _always_ enforcing a "1 package per import statement" policy.

```python
from foo import bar
from foo import baz
from foo import etc
```

You _can_ use the multiline approach:
```python
from foo import (
    bar,
    baz,
    etc,
)
```

though this starts to break down on certain edge cases..

What happens when there are two imports
```python
from foo import (
    bar,
    baz,
)
```

but then you remove one. Do you preserve the multiline format?
This ends up looking odd.
```python
from foo import (
    bar,
)
```

or do you condense back to single-line statements,
which unfortunately means muddying the `diff`,
increasing the likelihood of introducing inadvertent bugs/typos:

```diff
- from foo import (
-     bar,
-     baz,
- )
+ from foo import bazz
```

The same thing happens when what was previously a single import now includes multiple imports from the same package:
```diff
- from foo import bazz
+ from foo import (
+     bar,
+     baz,
+ )
```

A similar issue arises with multiple packages per import statement:
```diff
- from foo import bar, baz
+ from foo import baz
```

Meanwhile, "one-import per statement" ensures that:
- additions only ever add a single line
- deletions only ever remove a single line
- the multiline format is never used for a single import

```diff
from foo import bar
- from foo import baz
```

```diff
from foo import bar
+ from foo import baz
```

The noted downside, of course, is that it's a few more characters to use independent import statements, but if you aren't copy/paste/editing that already, you're doing it wrong :)